Samples/Sample XML/Candidate.xml

```diff
@@ -379652,6 +379652,81 @@
 			<GradeLevel>uri://ed-fi.org/GradeLevelDescriptor#Ungraded</GradeLevel>
 		</EPPProgramDegree>
 	</Candidate>
+  <CandidateIdentity>
+    <CandidateReference>
+		<CandidateIdentity>
+			<CandidateIdentifier>1001036</CandidateIdentifier>
+		</CandidateIdentity>
+	</CandidateReference>
+	<CandidateIdentificationSystem>uri://ed-fi.org/CandidateIdentificationSystemDescriptor#District</CandidateIdentificationSystem>
+		<EducationOrganizationReference>
+			<EducationOrganizationIdentity>
+				<EducationOrganizationId>255901</EducationOrganizationId>
+			</EducationOrganizationIdentity>
+		</EducationOrganizationReference>
+	<IdentificationCode>2141949</IdentificationCode>
+	<AssigningOrganizationIdentificationCode>SCOL-255901</AssigningOrganizationIdentificationCode>
+  </CandidateIdentity>
+    <CandidateIdentity>
+    <CandidateReference>
+		<CandidateIdentity>
+			<CandidateIdentifier>1001037</CandidateIdentifier>
+		</CandidateIdentity>
+	</CandidateReference>
+	<CandidateIdentificationSystem>uri://ed-fi.org/CandidateIdentificationSystemDescriptor#SSN</CandidateIdentificationSystem>
+		<EducationOrganizationReference>
+			<EducationOrganizationIdentity>
+				<EducationOrganizationId>255901001</EducationOrganizationId>
+			</EducationOrganizationIdentity>
+		</EducationOrganizationReference>
+	<IdentificationCode>837381631</IdentificationCode>
+	<AssigningOrganizationIdentificationCode>SCOL_255901001</AssigningOrganizationIdentificationCode>
```

Comment:
This pertains to all Identities:

AssigningOrganizationIdentificationCode is meant to be a more descriptive explanation of what type of code it is.  So the CandidateIdentificationSystemDescriptor might be a value of `local` and the AssigningOrganizationIdentificationCode could be `College of Ed id code` 

Descriptor could be `SSN`, AssigningOrganizationIdentificationCode might be `Social Security Number`

This does bring up a point I've thought before, we don't have a good enough description on what this field is meant to represent and is confusing, will take note to discuss updating the field description.

Additionally, AssigningOrganizationIdentificationCode is optional, you should have at least 1 entry in each identity type where this doesn't have a value